packages/dappmanager/src/modules/stakerConfig/setStakerConfig.ts

```diff
@@ -508,3 +520,70 @@ function setStakerConfigOnDb<T extends Network>({
       throw new Error(`Unsupported network: ${network}`);
   }
 }
+
+function checkLodestarMinVersions(
+  lodestarDnpName: string,
+  selectedExecClient: StakerItemOk<Network, "execution"> | undefined,
+  isSignerSelected: boolean | undefined,
+  web3signerPkg: InstalledPackageDataApiReturn | undefined
+): void {
+  if (!selectedExecClient) {
+    throw Error(`Execution client is not selected.`);
+  }
+
+  const lodestarMinVersions =
+    params.lodestarStakersMinimumVersions[lodestarDnpName];
+
+  if (!lodestarMinVersions) {
+    throw Error(
+      `Lodestar ${lodestarDnpName} minimum versions are not defined.`
+    );
+  }
+
+  const selectedExecClientVersion = selectedExecClient.data?.metadata.version;
+
+  //Throw error if execution client version cannot be fetched
+  if (selectedExecClientVersion === undefined) {
```

Comment:
Why not `if(!selectedExecClientVersion)` This way you check for falsy values also